src/locales/en.ts

```diff
@@ -332,6 +332,28 @@ export default {
       'Please be advised that the subDAO treasury has only been active for a few days since the Solana migration. Swapping subtokens to HNT may be affected by a limited HNT supply in the subDAO treasury. Please proceed with caution.',
     addRecipient: 'Add Recipient',
   },
+  defiTutorial: {
+    title: 'Safety Tips',
+    enterDefi: 'Enter DeFi',
```

Comment:
```suggestion
    enterDefi: 'Enter Ecosystem',
```

Feel like it's more an "enter the solana ecosystem" and less about just DeFi. It could be another realms page, or something like solana.playspokemon.xyz for example.

Reply:
Yeah agreed